pxtblocks/fields/field_ledmatrix.ts

```diff
@@ -217,11 +336,12 @@ export class FieldMatrix extends Blockly.Field implements FieldCustom {
 
             ev.stopPropagation();
             ev.preventDefault();
+            this.elt.blur();
```

Comment:
Can we get this by keeping stopPropagation but removing preventDefault?

Reply:
We have to keep `stopPropagation` to stop the SVG element with `tabindex="-1"` from taking focus on click. The code in the blur handler has been refactored and pulled out so it can be called here in a way where its intent is clear [07a5c6a](https://github.com/microbit-matt-hillsdon/pxt/pull/11/commits/07a5c6a5b1be6327fc4f00abffafc6d96c8ef43b)